src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -214,16 +215,17 @@ private void invalidateSlackAndReference() {
 
     public void updateSlackBusesAndReferenceBus() {
         if (slackBuses == null && referenceBus == null) {
-            SelectedSlackBus selectedSlackBus = slackBusSelector.select(busesByIndex, maxSlackBusCount);
-            slackBuses = selectedSlackBus.getBuses().stream()
-                    .filter(bus -> !excludedSlackBuses.contains(bus))
-                    .toList();
+            List<LfBus> selectableBus = busesByIndex.stream()
+                .filter(bus -> !excludedSlackBuses.contains(bus))
+                .toList();
```

Comment:
rename `selectableBus` to `selectableBuses` 

also wouldn't it beneficial performance wise to do this only if excludedSlackBuses is not empty ? (avoiding an unnecessary copy of the list)

Reply:
I guess for big network this can have an impact. Fixed.